backend/app/processing/server.py

```
@@ -85,18 +85,18 @@ async def process_query(user_id: str, chatbot_id: str, query: str, conversation_
         from processing.db_storage import DatabaseStorageManager
 
         # 1) Grab your Vanna instance
-        vn = vanna.get_instance(chatbot_id)
+        # vn = vanna.get_instance(chatbot_id)
         
-        # 2) Make sure it’s pointing at the right SQLite DB
-        storage_mgr = DatabaseStorageManager()
-        config = storage_mgr.load_config(chatbot_id)
-        vn.connect_to_sqlite(config["db_path"])
+        # # 2) Make sure it’s pointing at the right SQLite DB
```

Ruff (0.8.2)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)